### ISSUE TITLE
Permissions problem 

### ISSUE THREAD

**Issue by adelavega** [poldracklab/fitlins#74]
I'm getting a weird permissions problem. Fitlins is unable to create the output directory. I hadn't run into this issue before because in `neuroscout-cli` I extend your Dockerfile and switch the user to `root`. So the docker user is able to access my mounted volumes. But it seems with user `neuro` it's not able to.

    docker run -v /tmp/out:/out -v /home/zorro/datasets/test_datalad:/data --rm -it poldracklab/fitlins /data /out dataset -p /data/life/fmriprep

If I `chmod +666 /tmp/out` fitlins is able to run.

I could see the flip side of this, that by running with the user set to `root`, it may create files in the host filesystem with the incorrect permissions. Maybe this is why neurodocker doesn't use root?

**Comment by effigies** [comment]
Use `-u <your UID>` .

**Comment by adelavega** [comment]
Is this why neurodocker doesn't use root as default? I could see it making problematic files

**Comment by effigies** [comment]
Yes, that's the idea. There may be issues with acting within the container in `neuro`-owned directories, though. Might need to `chmod` rather than `chown` in the dockerfile.

**Comment by adelavega** [comment]
Yeah. When I did `chmod` in the host environment, that solved it. Didn't try from within though
